packages/core/src/pill/internal/PillCheckbox.css

```diff
@@ -13,13 +13,18 @@
   --uitkCheckbox-icon-tick-color: var(--uitk-taggable-foreground);
 }
 
-/* Backwards compat styling to match TK1 */
-.uitk-light .backwardsCompat.uitkPill .uitkPill-checkbox {
-  --uitkCheckbox-icon-stroke: var(--uitk-color-grey-100);
-  --uitkCheckbox-icon-tick-color: var(--uitk-color-grey-300);
+/* TK1 Backwards compat styling */
+
+.uitk-light {
+  --backwardsCompat-pill-checkbox-icon-stroke: var(--uitk-color-grey-100);  
+  --backwardsCompat-pill-checkbox-icon-tick-color: var(--uitk-color-grey-300);
+}
+.uitk-dark {
+  --backwardsCompat-pill-checkbox-icon-stroke: var(--uitk-color-grey-80);
+  --backwardsCompat-pill-checkbox-icon-tick-color: var(--uitk-color-grey-60);
 }
 
-.uitk-dark .backwardsCompat.uitkPill .uitkPill-checkbox {
-  --uitkCheckbox-icon-stroke: var(--uitk-color-grey-80);
-  --uitkCheckbox-icon-tick-color: var(--uitk-color-grey-60);
+.backwardsCompat.uitkPill .uitkPill-checkbox {
+  --uitkCheckbox-icon-stroke: var(--backwardsCompat-pill-checkbox-icon-stroke);
```

Comment:
This is an interesting one as its back compat for a nested component and so we need to use the API but this limits consumer overrides..

Reply:
yes, the trouble is this is a nested component (CheckboxIcon used in Pill) - so the API is used above for CheckboxIcon, and will always override the local var here even if it was set in backwards compat. so, we would have to also set the local var and ensure its higher specificity than the nested component itself, but atm we are using the API when overriding nested components, so would have to standardise that everywhere

Reply:
oops i accidentally deleted your comment @alycrys ! 

> "I think I'd prefer a local var. That's what we use on --formField-helperText-fontSize, don't we?"